### PR TITLE
To find the square root of a number

### PR DESCRIPTION
**Pull request by Shalini-jk**

### CODE REVIEW

File: Programs2/Squareroot.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import math
+a = int(input("enter a number whose square root u needed \n"))
+print(math.sqrt(a))
```
Comment (anurag-kqi) [inline Programs2/Squareroot.py]: idea is to find square root of a number without using math.sqrt function.

File: Programs2/Squareroot.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+a = int(input("enter a number whose square root You needed \n"))
+b = 1/2;
+c = (a ** b)
+print (c)
```
Comment (anurag-kqi) [inline Programs2/Squareroot.py:4]: Can you find another solution that does not use power function?

**Comment by Shalini-jk** [comment]
yes sir

File: Programs2/Prime.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+a = int(input("enter the number to check the number is prime or not ? \n"))
+if a > 1:
+    
+    for i in range(2, int(a / 2) + 1):
```
Comment (anurag-kqi) [inline Programs2/Prime.py:4]: You need to check number up to middle of that number or up to square root of that number?

**Comment by Shalini-jk** [comment]
sir it means i have to find the set of prime numbers between the numbers

**Pull request merged**